themes/default/content/docs/intro/concepts/stack.md

```diff
@@ -25,13 +25,16 @@ The stack name must be unique within a project. Stack names may only contain alp
 $ pulumi stack init staging
 ```
 
-If you are using Pulumi in your organization, by default the stack will be created in your user account. To target the organization, name the stack using `orgName/stackName`:
+There are a variety of available formats for the stackName, and the available formats differ between the [Service](/docs/intro/concepts/state#pulumi-service-backend) and [Self-Managed](/docs/intro/concepts/state#using-a-self-managed-backend) backends.
 
-```bash
-$ pulumi stack init broomllc/staging
-```
+| Stack Name Format               | Service     | Self-Managed |
+| ------------------------------- | ----------- | ------------ |
+| `stackName`                     | Identifies the stack `stackName` in the current user account or default organization, and the project specified by the nearest `Pulumi.yaml` project file.  | Identifies an unqualified stack `stackName` stored in the root of the backend. **Note**: The behaviour of this option may change in a future major version release to align with the service backend. |
+| `orgName/stackName`             | Identifies the stack `stackName` in the organization `orgName`, and the project specified by the nearest `Pulumi.yaml` project file. |  Not Supported - no notion of "organization" in the self-managed backend.  Separate self-managed backends can be used to manage seperate organizations. |
+| `orgName/projectName/stackName` | Identifies the stack `stackName` in the organization `orgName` and the project `proejctName`.  `projectName` must match the proejct specified by the nearest `Pulumi.yaml` project file. |  Not Supported - interpreted as `projectName/stackName`. |
```

Comment:
> Not Supported - interpreted as `projectName/stackName`.

Is the "interpreted as" accurate? When I try to do `pulumi stack init org/proj/stack` when logged into the filestate backend, it errors with:

```
error: could not parse stack reference 'org/proj/stack'
```

Instead of saying "interpreted as" should it say something about erroring?

Reply:
Ahh - indeed.  These two descriptions for the two "not supported" cases are inverted here.  Will fix.